auth/credentials.py

```diff
@@ -83,10 +120,100 @@ def validate_credentials(auth_username, auth_password_or_token):
         logger.debug("Found credentials header for robot %s", auth_username)
         try:
             robot = model.user.verify_robot(auth_username, auth_password_or_token)
+
             logger.debug("Successfully validated credentials for robot %s", auth_username)
+
             return ValidateResult(AuthKind.credentials, robot=robot), CredentialKind.robot
-        except model.InvalidRobotException as ire:
+        except model.DeactivatedRobotOwnerException as dre:
+            robot_owner, robot_name = parse_robot_username(auth_username)
+
+            try:
+                performer = model.user.lookup_robot(auth_username)
+            except User.DoesNotExist:
+                performer = None
+
+            logger.debug(
+                "Tried to use the robot %s for a disabled user: %s", (robot_name, robot_owner)
```

Comment:
Parens need to be removed here. It's passing in a single tuple for the first placement and nothing for the second.